src/Features/Units/Connector.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Connect to the current unit system from Redux store
+ */
+import React from "react"
+
+import { UnitSystem } from "Features/Units/types"
+
+import { useUnitSystem } from "./hooks"
+
+interface Props {
+  children(props: RenderProps): JSX.Element
```

Comment:
Codacy found an issue: ['JSX' is not defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)